source/administration/command-line-tools.rst

```diff
@@ -34,6 +34,11 @@ From the directory where the Mattermost platform is installed, a
 .. contents::
     :backlinks: top
 
```

Comment:
If you add :local: to this then the page title won't be included in the contents. For example:

```
.. contents::
  :backlinks: top
  :local:
```